packages/ldes-manager/lib/routes/Orchestrator.route.ts

```diff
@@ -3,12 +3,33 @@ import fp from 'fastify-plugin';
 import { Connector } from '../models/Connector.model';
 import { Orchestrator } from '../models/Orchestrator.model';
 import sequelize from '../../bin/orm/sequelize';
+import { ConnectorOrchestrator } from '../models/ConnectorOrchestrator.model';
 
 interface orchestratorParams {
   id: number;
 }
 
+interface orchestratorAttrs {
+  ldes_uri: string;
+  name: string;
+  connectorId: number;
+  polling_interval: number;
+}
+
+const orchestratorBodyJsonSchema = {
+  type: 'object',
+  required: ['name', 'ldes_uri', 'connectorId', 'polling_interval'],
+  properties: {
+    name: { type: 'string' },
+    ldes_uri: { type: 'string' },
+    connectorId: { type: 'number' },
+    polling_interval: { type: 'number' },
```

Comment:
If possible, maybe also use snake_caase for the connectorId. I think I know why it's camel case, but yeah, for and end user one casing type would be easier to work with

Reply:
Done !